### ISSUE TITLE
[starter-library] "Submit a Starter" button is broken

### ISSUE THREAD

**Issue by Manoz** [gatsbyjs/gatsby#8968]
## Description

The **Submit a Starter** button is a bit broken on mobile version.

### Steps to reproduce

Take a look at the [starters](https://www.gatsbyjs.org/starters/) page on your smartphone or on mobile resolution with your dev tools.

### Expected result

I have no idea. What should we do? Since the block with the title and the search form are sticky on scroll what should we do? Hide the button on mobile?

![screenshot-2018-10-09_23-12-53](https://user-images.githubusercontent.com/1554207/46699123-29108f80-cc19-11e8-9b8b-d0b75b1c20c3.png)

### Actual result

![submit-a-starter](https://user-images.githubusercontent.com/1554207/46698967-bc959080-cc18-11e8-820b-9f1ffeb30eff.png)

### Environment

```
System:
  OS: Windows 10
  Mobile OS: iOS 12.1 Public beta 3
Binaries:
  Yarn: 1.9.4
  npm: 6.3.0
Browsers:
  Google Chrome: 69.0.3497.100
  Google Chrome on iPhone (latest version)
  Safari on iOS (latest version)
```

**Comment by lmcjt37** [comment]
Happy to pick this up? 

How likely is it that anyone would want to 'submit a starter' via their mobile device? I would suggest this be hidden for mobile

**Comment by kakadiadarpan** [comment]
In my opinion, we should fix the CSS.

@pieh what do you think about this?

**Comment by Manoz** [comment]
@lmcjt37 there's a PR already :) The button is actually a link to the documentation page so it might be useful on mobile too

**Comment by tanvibhakta** [comment]
Here's the PR for those interested:
https://github.com/gatsbyjs/gatsby/pull/8969

It looks like discussion has stalled on that thread. Can we add a claimed label to this bug as well?

**Comment by Manoz** [comment]
@tanvibhakta This is not part of the Hacktoberfest but why not :p

**Comment by amberleyromo** [comment]
Fixed in #10567.
